front/docker/Dockerfile.devel

```diff
@@ -19,4 +19,4 @@ WORKDIR /app
 # Start the app
 COPY docker/dev-entrypoint.sh /
 ENTRYPOINT ["/dev-entrypoint.sh"]
-CMD ["sh", "-c", "id && yarn install && exec yarn start-container"]
+CMD ["sh", "-c", "id && rm -rf node_modules && npm install && exec npm run start-container"]
```

Comment:
I think `rm -rf node_modules` is no longer necessary now.

Reply:
yep, missed that, it's done